packages/vitest/src/node/config.ts

```diff
@@ -212,7 +212,9 @@ export function resolveConfig(
         ?? resolve(resolved.root, file),
     ),
   )
-  resolved.coverage.exclude.push(...resolved.setupFiles.map(file => relative(resolved.root, file)))
+  const exclude = toArray(resolved.coverage.exclude || [])
```

Comment:
As I said in https://github.com/vitest-dev/vitest/issues/3450#issuecomment-1567149596, we should guard all CLI option that can be passed down as array with such a guard

Reply:
Refer to the [UserConfig](https://github.com/vitest-dev/vitest/blob/2deb70abcac02d96b1d38c1300e5f70a32e9dc1c/packages/vitest/src/types/config.ts#L625) type for option guard, I don’t know if there is any omission.

Reply:
Only options in CLI are respected:

https://github.com/vitest-dev/vitest/blob/952b5be6ee1cda5016b33d233a6abf7620745e2c/packages/vitest/src/node/cli.ts#L12